test/functional/python/test_cell.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+from spglib.spglib import _expand_cell
+
+
+def test_expand_cell():
+    with pytest.raises(TypeError) as e:
+        _expand_cell(None)
+    assert e.value
```

Comment:
Check the actual message here

Reply:
Do we need to duplicate the error message twice? Is it common practice?

Reply:
Not the whole message, just `in`-checking is enough.

With warnings it definitely needs to, but for exceptions it depends. It does help guarantee we are catching our exceptions and not other built-in.